src/univalue/.travis.yml

```diff
@@ -26,6 +25,7 @@ addons:
       - pkg-config
 
 before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew uninstall libtool; brew install libtool; fi
```

Comment:
The upstream issue (https://github.com/jgarzik/univalue/pull/28) is of a somewhat discutable quality. Another reference to the outside source. I just want to know what is the source of the problem that is solved by uninstall/install round...